pom.xml

```diff
@@ -50,6 +50,10 @@
             <artifactId>opencsv</artifactId>
             <version>5.7.1</version>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.datatype</groupId>
+            <artifactId>jackson-datatype-jsr310</artifactId>
+        </dependency>
```

Comment:
Could you check if this dependency is needed? I remember using this dependency for datetime but finding out it is not needed.

Reply:
I think we don't really need it. I just removed it!